infrastructure/docker/Dockerfile-gateway

```diff
@@ -15,6 +15,7 @@ RUN apt-get -y update &&\
 USER 0
 COPY gateway/requirements.txt .
 RUN pip install -r requirements.txt --no-cache-dir
+RUN pip install opentelemetry-distro opentelemetry-exporter-otlp
```

Comment:
are these not picked up for install via the previous step?

Reply:
Thanks! I forgot to remove this line after I updated the requirements.txt